src/brokers/comdirect.js

```diff
@@ -389,112 +389,97 @@ export const canParseDocument = (pages, extension) => {
     firstPageContent.every(
       line => !line.includes(onvistaIdentificationString)
     ) &&
-    (isBuy(firstPageContent) ||
-      isSell(firstPageContent) ||
-      isDividend(firstPageContent) ||
-      isTaxinfoDividend(firstPageContent) ||
-      detectedButIgnoredDocument(firstPageContent))
+    getDocumentType(firstPageContent) !== undefined
   );
 };
 
-const parseData = textArr => {
-  let type,
-    date,
-    time,
-    isin,
-    wkn,
-    company,
-    shares,
-    price,
-    amount,
-    fee = 0,
-    tax = 0,
-    fxRate,
-    foreignCurrency;
+const parseData = (textArr, type) => {
+  let activity = {
+    broker: 'comdirect',
+    type: getDocumentType(textArr),
```

Comment:
Type is already parsed by function.

```suggestion
    type,
```